src/main/java/com/google/maps/model/AddressType.java

```diff
@@ -160,12 +163,45 @@
    */
   POINT_OF_INTEREST("point_of_interest"),
 
+  // In addition to the above, address components may include the types below.
```

Comment:
this comment could use reworking

"The values below are address components, which are <blah>".